test/ts/customRunner.ts

```diff
@@ -16,7 +16,7 @@ type Options =
 export const run = async (options: Options): Promise<boolean> => {
   const opts: Omit<RunnerOption, 'direction'> & Options = {
     migrationsTable: 'migrations',
-    dir: resolve(__dirname, 'migrations'),
+    dir: resolve('test', 'ts', 'migrations'),
```

Comment:
**thought (if-minor):** this is just a test-file and does not need to be changes until `package.json` is set to `"type": "module"`

Reply:
True, but we are already trying to remove everything that depends on `__dirname`. I don't see why leaving it for later, but I can remove the change if you think it's better. 🤝

Reply:
It is more like putting focus on one thing (fixing the source code for ESM).
But I might see now what you did here.

So... I follow the strategy to squash-merge every PR and using its title as commit message, so maybe extract this change into its own PR with title `test(runner): <message>`